src/blocks/time.rs

```diff
@@ -53,9 +54,11 @@ pub enum Timezone {
 }
 
 pub async fn run(config: Config, mut api: CommonApi) -> Result<()> {
-    // "next_timezone" changes the current displayed timezone to the timezone next in the list.
-    api.set_default_actions(&[(MouseButton::Left, None, "next_timezone")])
-        .await?;
+    api.set_default_actions(&[
+        (MouseButton::WheelDown, None, "next_timezone"),
+        (MouseButton::WheelUp, None, "prev_timezone"),
+    ])
+    .await?;
```

Comment:
This is a breaking change

Reply:
apologies, i can change it to left for `next_timezone` and right for `prev_timezone`